swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+
+    def __init__(self, id=None, condition=0, age=0, fabric="Unknown"):
+        super().__init__(id, condition, age)
+        self.fabric = fabric
+
+    def get_category(self):
+        return "Clothing"
+    
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}. It is made from {self.fabric} fabric."
```

Comment:
👀 Notice the similarity of the first sentence of the description with the description in `Item`'s `__str__`. They differ only in the name of the class being displayed. How could we modify `Item`'s `__str__` so that we could also use it in the child class implementations?